src/help/components/contextual/ContextualHelpContainer.tsx

```diff
@@ -0,0 +1,29 @@
+import { Loader } from 'franklin-sites';
+import ErrorHandler from '../../../shared/components/error-pages/ErrorHandler';
+import { help as helpUrl } from '../../../shared/config/apiUrls';
+import useDataApiWithStale from '../../../shared/hooks/useDataApiWithStale';
+import { HelpEntryResponse } from '../../adapters/helpConverter';
+import { HelpEntryContent } from '../entry/Entry';
+
+const ContextualHelpContainer = ({ articleId }: { articleId: string }) => {
+  const { data, loading, error, status, progress, isStale } =
+    useDataApiWithStale<HelpEntryResponse>(helpUrl.accession(articleId));
+
+  if (loading && !data) {
+    return <Loader progress={progress} />;
+  }
+
+  if (error || !data) {
+    return <ErrorHandler status={status} />;
+  }
+
+  return (
+    <HelpEntryContent
+      data={data}
+      isStale={isStale}
+      handleClick={() => console.log('click!')}
+    />
+  );
+};
```

Comment:
You can start using `SlidingPanel` rightaway, size="small". I'll modify the onClose handler in Franklin to make it optional and so that it gives in the callback the reason for which it has been called. So you could have here something like:
```
onClose={(reason) => {
  if (reason === 'button' || reason === 'navigation'){
    setDisplay(false)
  }
}}
```
And ignore the other reason why it's being called (click outside)